packages/dd-trace/src/telemetry/index.js

```diff
@@ -76,9 +76,30 @@ function createAppObject () {
 }
 
 function createHostObject () {
+  let osName = os.type()
+  if (osName === 'Linux' || osName === 'Darwin'){
+    return {
+      hostname: os.hostname(),
+      os: osName,
+      architecture: os.arch(),
+      kernel_version: os.version(),
+      kernel_release: os.release(),
+      kernel_name: osName
+    }
+  }
+
+  else if (osName === 'Windows_NT') {
+    return {
+      hostname: os.hostname(),
+      os: osName,
+      os_version: os.version(),
+      architecture: os.arch()
```

Comment:
nitpick: switching the order of the prop for shadowclass optimization
```suggestion
      architecture: os.arch(),
      os_version: os.version()
```